packages/fake-data/src/fakeUsers.ts

```diff
@@ -51,7 +48,6 @@ type GeneratedUser = User & {
 const generateUser = (
   departments: Department[],
   classifications: Classification[], // all classifications
-  genericJobTitles: GenericJobTitle[], // all generic job titles
```

Comment:
While looking at `fakeUsers.ts` I noticed this was unused and didn't actually get used in the function return. 
Probably left floating around after #8000

Reply:
Are we not using `genericJobTitle` at all anymore? I noticed a lot of unused refs to it while converting tests to TS.

Reply:
Pretty much, ya, #7578